.dockerignore

```diff
@@ -1,17 +1,32 @@
 node_modules
+**/__pycache__
+*.pyc
+
 Dockerfile*
 docker-compose*
 .dockerignore
 .git
+.github/
 .vscode
+deployment/
+docs/
+docker/
 env/
 local.py
 /doc/_build/
+/nlx
 
 /htmlcov
 .coverage
+# tests and test assets
+src/**/tests/**/*.py
```

Comment:
ehm yeah, good one. I think I first had `test_*.py` so that we don't accidentally ignore `conf/test.py` (if it would exist), but just ignoring the `tests` directory completely should do. If that breaks anything, then someone is using code that should only be used in tests.